### PR TITLE
Master torque error debug

### PR DESCRIPTION
**Pull request by p3jawors**
Changed timeout from 201ms to 501ms to minimize unexpected torque errors

### CODE REVIEW

File: abr_jaco2/interface/jaco2_rs485.cpp [hunk 1]
```diff
@@ -596,11 +601,12 @@ void Jaco2::ProcessFeedback() {
                 PrintError(ii, current_motor);
                 // in case of an error, go on to the next packet
 
-                for (int jj = 0; jj < 6; jj++) {
-                    clear_error_message[jj].DataLong[1] =
-                        ((unsigned long) feedback_message[ii].DataLong[0]);
-                }
-                SendAndReceive(clear_error_message, false);
+                // preset to 0 to clear all errors
```
Comment (studywolf) [inline abr_jaco2/interface/jaco2_rs485.cpp]: this code is commented, should it be removed?

File: abr_jaco2/interface/jaco2_rs485.cpp [hunk 2]
```diff
@@ -680,12 +686,12 @@ void Jaco2::ProcessFeedback() {
 // Process the error message, if it's an actual error then print out
 // information (it is possible that an error message is not an actual error)
 void Jaco2::PrintError(int index, int current_motor) {
-    if (feedback_message[index].DataLong[1] != 0) {
-        char buffer [100];
-        sprintf(buffer, "Message: %u %s for motor %d", feedback_message[index].DataLong[1],
-                error_message[feedback_message[index].DataLong[1]].c_str(), current_motor);
-        log_msg(4, buffer);
-    }
+    //if (feedback_message[index].DataLong[1] != 0) {
```
Comment (studywolf) [inline abr_jaco2/interface/jaco2_rs485.cpp]: commented -> removed?

File: abr_jaco2/interface/jaco2_rs485.cpp [hunk 3]
```diff
@@ -680,12 +686,12 @@ void Jaco2::ProcessFeedback() {
...
+    char buffer [100];
+    sprintf(buffer, "Message: %u %s for motor %d", feedback_message[index].DataLong[1],
+            error_message[feedback_message[index].DataLong[1]].c_str(), current_motor);
+    log_msg(4, buffer);
+    //}
```
Comment (studywolf) [inline abr_jaco2/interface/jaco2_rs485.cpp]: removed?

File: abr_jaco2/interface/jaco2_rs485.cpp [hunk 4]
```diff
@@ -205,7 +206,7 @@ Jaco2::Jaco2(int a_display_error_level) {
         safety_message[ii].SourceAddress = SOURCE_ADDRESS;
         safety_message[ii].DestinationAddress = JOINT_ADDRESS[ii];
         safety_message[ii].DataFloat[0] = MAX_TORQUE[ii]; // Nm maximum torque
-        safety_message[ii].DataFloat[1] = 1.0; //0.75 safety factor
+        safety_message[ii].DataFloat[1] = 1.0; //0.75 safety factor, fraction of max speed before error sent, 1=off
```
Comment (studywolf) [inline abr_jaco2/interface/jaco2_rs485.cpp]: less than 80 chars?

Comment (p3jawors) [reply abr_jaco2/interface/jaco2_rs485.cpp]: Done: split up to two lines

File: abr_jaco2/interface/jaco2_rs485.cpp (hunk 1 above)
Comment (p3jawors) [reply abr_jaco2/interface/jaco2_rs485.cpp]: removed

File: abr_jaco2/interface/jaco2_rs485.cpp (hunk 2 above)
Comment (p3jawors) [reply abr_jaco2/interface/jaco2_rs485.cpp]: removed

File: abr_jaco2/interface/jaco2_rs485.cpp (hunk 3 above)
Comment (p3jawors) [reply abr_jaco2/interface/jaco2_rs485.cpp]: removed

**Pull request merged**